cmd/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	_ "github.com/agneborn98/Arrow-Flight-ModelarDB-Output-Plugin/plugins/output/flight"
+
+	"github.com/influxdata/telegraf/plugins/common/shim"
+)
+
+func main() {
+
+	var pollInterval = flag.Duration("poll_interval", 1*time.Second, "how often to send metrics")
```

Comment:
I'm going to need some elaboration. 

Should I just write what units are possible to enter as arguments? (Anything can be entered here) 

Or do you mean what the default duration is?

Reply:
I am not sure how this works. Do you not just enter a number? If just entering a number it can be hard to know if entering "1" means 1 second or 1 millisecond for example. But if you enter an actual duration then this might not be an issue.